examples/sdk/browser/src/index.ts

```diff
@@ -14,6 +15,40 @@ const client = BacktraceClient.initialize({
     },
 });
 
+interface DemoState {
+    count: number;
+}
+
+const initialState: DemoState = {
+    count: 0,
+};
+
+const counterSlice = createSlice({
+    name: 'Redux-Demo',
+    initialState,
+    reducers: {
+        setCount(state, action) {
+            state.count = action.payload;
+        },
+    },
+});
+
+const { setCount } = counterSlice.actions;
+
+const store = configureStore({
```

Comment:
isn't this a react specific library? Do we need to add it to the package.json?

Reply:
I'm not quite sure what you mean. Redux.js is UI/framework agnostic (but it does tend to be used the most with react). All of this redux functionality is included in `@reduxjs/toolkit`. Is that what you mean?

Reply:
This is what I mean - we do not have @reduxjs/toolkit inside the package.json definition

Reply:
It's there. It's a dependency in the browser example package.json.